### DISCUSSION TITLE
cannot access previous account and app

### DISCUSSION THREAD

**Discussion by thevegasmovement** [Budibase/budibase#7035] [Help]
hi, I had an app built on the older version and had been given like 10k automations in my account for free and I cannot access this app or the automation (10k) that had been given to me (via a discord conversation when i needed help). 

Does this mean that I have to build my app all over again? and how can I get access to these 10k automations? It was under realcreditkv@gmail.com

**Comment by PClmnt** [comment]
Hi @thevegasmovement. Can you be a little more specific about the issue? Are you recieving an error when you try to access the design section of your app? What was your tenant name? i.e` {tenant}.budibase.app`. Or are you self hosted?

**Reply by thevegasmovement** [reply]
hi I was not self-hosting. I was using the app on my pc - I am now using the web version to build a new one. I cannot recall what the tenant name was. I can re-build the app (which I am now), but wanted to see how I am able to get the automations that were supposed to have been added to my account (there's no way for me to check on that). If I have to, I will go through my discord conversations to find where it had  been given to me.
